docs/asciidoc/modules/ROOT/pages/ml/vertexai.adoc

```diff
@@ -111,7 +111,7 @@ yield value
 [%autowidth, opts=header]
 |===
 |name | description
-| messages | List of maps of instructions with `{author:"bot|user", content:"text"}`
+| messages | List of maps of instructions with `{author:"bot user", content:"text"}`
```

Comment:
The correct formatting should be with `\|`, which return it `{author:"bot|user", content:"text"}`, which means `"author": "bot" OR "user"`

```suggestion
| messages | List of maps of instructions with `{author:"bot\|user", content:"text"}`
```